docs/content/installation/csi_ug_requirements.md

```diff
@@ -27,43 +27,69 @@ For IBM Cloud® Satellite users, see [cloud.ibm.com/docs/satellite](https://clou
 
-   For more information and instructions, see the Kubernetes blog post, [Kubernetes 1.20: Kubernetes Volume Snapshot Moves to GA](https://kubernetes.io/blog/2020/12/10/kubernetes-1.20-volume-snapshot-moves-to-ga/).
+2. Configure storage system connectivity.
 
-   Install both the Snapshot CRDs and the Common Snapshot Controller once per cluster.
+    1.  Define the host of each Kubernetes node on the relevant storage systems with the valid WWPN (for Fibre Channel) or IQN (for iSCSI) of the node.
 
-   The instructions and relevant yaml files to enable volume snapshots can be found at: [https://github.com/kubernetes-csi/external-snapshotter#usage](https://github.com/kubernetes-csi/external-snapshotter#usage)
+    2.  For Fibre Channel, configure the relevant zoning from the storage to the host.
 
-4. Configure storage system connectivity.
+    3. Ensure proper connectivity.
 
-    1.  Define the host of each Kubernetes node on the relevant storage systems with the valid WWPN (for Fibre Channel) or IQN (for iSCSI) of the node.
+3. **For RHEL OS users:** Ensure that the following packages are installed.
 
-    2.  For Fibre Channel, configure the relevant zoning from the storage to the host.
+    If using RHCOS or if the packages are already installed, this step may be skipped.
+
+       - sg3_utils
+       - iscsi-initiator-utils
+       - device-mapper-multipath
+       - xfsprogs (if XFS file system is required)
 
+4. (Optional) If planning on using volume snapshots (FlashCopy® function), enable support on your Kubernetes cluster.
+
+   For more information and instructions, see the Kubernetes blog post, [Kubernetes 1.20: Kubernetes Volume Snapshot Moves to GA](https://kubernetes.io/blog/2020/12/10/kubernetes-1.20-volume-snapshot-moves-to-ga/).
+
+   Install both the Snapshot CRDs and the Common Snapshot Controller once per cluster.
+
+   The instructions and relevant YAML files to enable volume snapshots can be found at: [https://github.com/kubernetes-csi/external-snapshotter#usage](https://github.com/kubernetes-csi/external-snapshotter#usage)
+
+5. (Optional) If planning on using volume replication (remote copy function), enable support on your orchestration platform cluster and storage system.
+    
+    1. To enable support on your Kubernetes cluster, install the following replication CRDs once per cluster.
+
+        ```
+        curl -O https://raw.githubusercontent.com/csi-addons/volume-replication-operator/v0.1.0/config/crd/bases/replication.storage.openshift.io_volumereplicationclasses.yaml
+        kubectl apply -f ./replication.storage.openshift.io_volumereplicationclasses.yaml
+        
+        curl -O https://raw.githubusercontent.com/csi-addons/volume-replication-operator/v0.1.0/config/crd/bases/replication.storage.openshift.io_volumereplications.yaml
+        kubectl apply -f ./replication.storage.openshift.io_volumereplications.yaml
```

Comment:
`v0.1.0` -> `v0.2.0`

Reply:
@oriyarde  hould it be changed in both of the curl commands?
This is why full reviews are important before approval and my release.
I'll open a ticket and new branch to make an update to the documentation.

Reply:
`curl -O https://raw.githubusercontent.com/csi-addons/volume-replication-operator/v0.2.0/config/crd/bases/replication.storage.openshift.io_volumereplicationclasses.yaml`

`curl -O https://raw.githubusercontent.com/csi-addons/volume-replication-operator/v0.2.0/config/crd/bases/replication.storage.openshift.io_volumereplications.yaml`

Reply:
was done [here](https://github.com/IBM/ibm-block-csi-driver/pull/390/files#diff-c556cca7f72801b0345c2f6308493cd83c4e4e9e65b82d3e3c8523ebf8915ced). resolving